dockers/docker-ptf/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         python-dev          \
         wget                \
         cmake               \
+	isc-dhcp-server	    \
```

Comment:
Okay, that's an easy change. I will install and start dhcp server from playbook instead.